.gitignore

```diff
@@ -1,6 +1,7 @@
 *.egg-info/
 *.pyc
 .coverage
+.idea/
```

Comment:
This looks like a user specific thing unrelated to the project. It best fits a [global `.gitignore` file](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/), not the project's `.gitignore` file.

```suggestion
```